tools/check_format.py

```diff
@@ -36,16 +39,48 @@ def checkNamespace(file_path):
   return True
 
 
+# To avoid breaking the Lyft import, we just check for path inclusion here.
+def whitelistedForProtobufDeps(file_path):
+  return len([prefix in file_path for prefix in GOOGLE_PROTOBUF_WHITELIST]) > 0
```

Comment:
nit: s/prefix/path_segment 

Right? (Just making sure I'm reading this correctly).